src/SprykerEco/Zed/UnzerApi/Business/Api/Response/Mapper/AuthorizeResponseMapper.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Zed\UnzerApi\Business\Api\Response\Mapper;
+
+use Generated\Shared\Transfer\UnzerApiMarketplaceAuthorizeResponseTransfer;
+use Generated\Shared\Transfer\UnzerApiMessageTransfer;
+use Generated\Shared\Transfer\UnzerApiResponseTransfer;
+use SprykerEco\Zed\UnzerApi\Business\Api\Request\UnzerApiRequestConstants;
+
+class AuthorizeResponseMapper implements UnzerApiResponseMapperInterface
+{
+    /**
+     * @param array $responseData
+     * @param \Generated\Shared\Transfer\UnzerApiResponseTransfer $unzerApiResponseTransfer
+     *
+     * @return \Generated\Shared\Transfer\UnzerApiResponseTransfer
+     */
+    public function mapResponseDataToUnzerApiResponseTransfer(
+        array $responseData,
+        UnzerApiResponseTransfer $unzerApiResponseTransfer
+    ): UnzerApiResponseTransfer {
+        $unzerApiMarketplaceAuthorizeResponseTransfer = (new UnzerApiMarketplaceAuthorizeResponseTransfer())
```

Comment:
I think `->fromArray()` will work here as well

Reply:
The same goes for creating transfers below.

Reply:
Check other mappers too.

Reply:
I suggest to use explicit setters here to not be linked to response array keys and also to see the real structure